src/components/Header.tsx

```diff
@@ -3,9 +3,10 @@ import './Header.css';
 
 export const Header = () => {
   return (
-    <header>
+    <header role='header'>
       <Link className='' to={'/'}>
         <img
+          data-testid='header'
```

Comment:
Looks good. I see in your test that you are not using this ´´data-testid='header'´´ which is good, so you could most likely remove it :).

Great work!

Reply:
Oh yeah that's right, it's fixed now, Thanks! :)